### PR TITLE
updated Pr template for screenshots

### LINKED ISSUE #1048: Make contribution template clearer for contributors
In particular, the line:

"Please include screenshots, gifs, etc."

Has at times been interpreted by contributors to mean of the code itself. It would be great to make it clearer that we're looking for all changes of any size to include a screenshot and/or video of the code actually functioning.

### PR DESCRIPTION
**Pull request by dotslashbit**
## Description

closes #1048 

## Changes

Modified the screenshots section to be more specific

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Please describe the changes in detail.
 
 ## Screenshots
 
-Please include screenshots, gifs, etc.
+Upload concise visuals (screenshots, gifs, etc) showing the output and impact of changes. Avoid sharing code screenshots or videos.
```
Comment (MaxGhenis) [inline .github/PULL_REQUEST_TEMPLATE.md:11]: videos are fine, even encouraged! some of us use loom extensively. code segments are fine too if they help illustrate the solution, they're just not what we mean by screenshots.

Comment (dotslashbit) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: How about this:
## Screenshots and Visuals

Upload concise visuals (screenshots, gifs, etc) demonstrating the output and impact of changes. Feel free to use videos, especially if utilizing tools like Loom. While code segments are acceptable if they enhance understanding, avoid sharing full code screenshots or videos.

Comment (anth-volk) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: I think we could still soften the last bit, as screenshots of the code itself are fine, just usually not necessary. What about something like this:

Please include concise visuals (videos, screenshots, etc.) demonstrating the code in action and the impact of changes. Many on the team use a free tool called [Loom](https://www.loom.com) to capture and share short clips, but feel free to utilize any tool you'd like. Screenshots of the code itself can also be helpful, but are usually not necessary.

Comment (dotslashbit) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: This is good, actually I was a bit hesitant on adding a long statement like this, but if its fine then I think this is a good one, so should I replace your statement in my PR?

Comment (anth-volk) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: If you don't mind, I think it gets at what we're looking for a bit more concisely. That said, it's like 70% your statement, anyway haha

Comment (dotslashbit) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: I also think that your statement is very clear, so if you want me to replace my statement with yours, then let me know I'll modify this in the PR

Comment (anth-volk) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: I think it would be best

**Comment by dotslashbit** [comment]
@anth-volk I have updated the text, please let me know if there's anything needed to be done

**Review by anth-volk — APPROVED** [review]
This looks great, thanks so much

**Pull request merged**